constellations/gemini-user/src/afh/afh.js

```diff
@@ -0,0 +1,152 @@
+                sessions: afh,
+            });
+        });
+    }
+
+    openSubject = (subjectName) => {
+        this.setState({
+            currentTab: subjectName,
+        });
+    };
+
+    render() {
+        let currentSub = this.state.sessions.filter(
+            (session) => session.subject == this.state.currentTab
+        );
+
+        let showSessions = currentSub.map((row, index) => {
+            let sessionDate = moment(row.date).format("M/D/YYYY dddd");
+            return (
+                <div
+                    className={
+                        this.state.isActive == "true"
+                            ? "sessions-list-active"
+                            : "sessions-list"
+                    }
+                    key={index}>
+                    <div className="sessions-checkbox">
+                        <CheckboxInput
+                            key={index}
+                            row={row}
+                            onChangeCheckbox={this.onChangeCheckbox}
+                        />
+                    </div>
+
+                    <div className="session-details">
+                        {sessionDate} {row.timeString} <br />
+                        {row.title} {row.notes} <br /> {row.locationId}
+                    </div>
+                </div>
+            );
+        });
+
+        return (
+            <div id="view-afh">
+                <h1>Ask for Help</h1>
+                <div className="description">
+                    We provide free sessions for students to ask for additional
+                    assistance on any of our program subjects. Please fill the
+                    form to let us know you are coming. You must be registered
+                    with one of our programs to attend.
+                </div>
+
+                <h1>Ask for Help Sessions by Subject</h1>
+                <div className="tab">
+                    <button
+                        className={
+                            this.state.currentTab == "Math" ? "active" : ""
+                        }
+                        onClick={() => this.openSubject("Math")}>
+                        Math
+                    </button>
+                    <button
+                        className={
+                            this.state.currentTab == "English" ? "active" : ""
+                        }
+                        onClick={() => this.openSubject("English")}>
+                        English
+                    </button>
+                    <button
+                        className={
+                            this.state.currentTab == "Computer Programming"
+                                ? "active"
+                                : ""
+                        }
+                        onClick={() =>
+                            this.openSubject("Computer Programming")
+                        }>
+                        Computer Programming
+                    </button>
```

Comment:
These 3 buttons are all kind of the same. Just with a few values that are different which we can pass in as component props.

So maybe something like this:
```
<TabButton currentTab={this.state.currentTab}/>
```

When you pass in this.state.currentTab ("math", "english", or "computer"), With the subjectDisplayNames I mentioned above, you also have access to what to present to the user!
`subjectDisplayNames[this.props.currentTab]`